src/ApiBundle/Entity/User.php

```diff
@@ -48,6 +49,17 @@ class User extends BaseUser
     protected $dob;
 
     /**
+    * @var string $image
+    * @Assert\File(
+    *     maxSize = "1024k",
```

Comment:
As discussed, User (who is signing up) should be able add/ remove or update user profile image; not the admin only.